app/controllers/activities_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ActivitiesController < ApplicationController
+	include ActivitiesHelper
```

Comment:
Now I understand the problem. I guess you haven't enabled `convert tabs to spaces` in sublime settings.